### ISSUE TITLE
Is that possible to add Japanese-Chinese Dictionary?

### ISSUE THREAD

**Issue by jonathonyan** [FooSoft/yomichan#1911]
As the title

**Comment by shoui520** [comment]
Yes you can find one in my Yomichan dictionaries collection https://learnjapanese.link/dictionaries

**Comment by jonathonyan** [comment]
Which dictionary is from Japanese to Chinese?  I only found one which is from Chinese to Japanese.

**Comment by wildwestrom** [comment]
https://drive.google.com/drive/folders/1cQprswMScgMx7UDxfN-BsoTlPaDGheS6 @jonathonyan
